resources/views/filament-monaco-editor.blade.php

```diff
@@ -169,4 +169,32 @@ class="fme-wrapper"
         </div>
     </div>
 
+    @script
```

Comment:
Could also be wrapped in the `@verbatim` directive, but that gives less IDE support when using @ signs in regex. https://laravel.com/docs/11.x/blade#the-at-verbatim-directive

Reply:
Best IDE support would be if it was used in the standalone JS files, but they don't seem to be loaded.